actors/miner/src/policy.rs

```diff
@@ -114,15 +123,36 @@ pub fn quality_for_weight(
     let scaled_up_weighted_sum_space_time: SectorQuality =
         weighted_sum_space_time << SECTOR_QUALITY_PRECISION;
 
-    scaled_up_weighted_sum_space_time
+    let quality = scaled_up_weighted_sum_space_time
         .div_floor(&sector_space_time)
-        .div_floor(&QUALITY_BASE_MULTIPLIER)
+        .div_floor(&QUALITY_BASE_MULTIPLIER);
+    // Apply SDM
+    sdm_scale(quality, duration)
+}
+
+/// Scale `value` by the sector duration multiplier for `duration`.
+pub fn sdm_scale<T>(mut value: T, duration: ChainEpoch) -> T
+where
+    T: MulAssign<ChainEpoch> + DivAssign<ChainEpoch>,
+{
+    if duration > SECTOR_DURATION_MULTIPLIER_START {
+        if duration > SECTOR_DURATION_MULTIPLIER_END {
+            log::error!("sector lifetime {duration} exceeded maximum expected lifetime {SECTOR_DURATION_MULTIPLIER_END}");
+            value *= 2;
```

Comment:
What's this constant `2`? Is it related to MAX_WEIGHT_MULTIPLIER?

Reply:
It's the max sector duration multiplier. I'll move it to a named constant.